packages/store/package.json

```diff
@@ -31,6 +31,7 @@
   "homepage": "https://github.com/Agoric/agoric-sdk#readme",
   "dependencies": {
     "@agoric/assert": "^0.6.0",
+    "@agoric/time": "^0.3.2",
```

Comment:
why?

Reply:
No longer needed.

Reply:
I undid my reversion. Without it (notice that it's also present in `devDependencies`) tests don't build.